homeassistant/components/wyoming/satellite.py

```diff
@@ -126,12 +135,23 @@ async def on_stopped(self) -> None:
 
     # -------------------------------------------------------------------------
 
+    def _send_pause(self) -> None:
+        """Send a pause message to satellite."""
+        if self._client is not None:
```

Comment:
Shouldn't this check happen before calling this method ?

Reply:
It's called from multiple places. I could duplicate the check at the call sites if that would be better.